src/Temporalio/Bridge/include/temporal-sdk-bridge.h

```diff
@@ -358,6 +358,11 @@ typedef struct WorkerOptions {
   uint32_t max_concurrent_workflow_task_polls;
   float nonsticky_to_sticky_poll_ratio;
   uint32_t max_concurrent_activity_task_polls;
+  bool nondeterminism_as_workflow_fail;
+  /**
+   * This is expected to be a newline-delimited list
+   */
+  struct ByteArrayRef nondeterminism_as_workflow_fail_for_types;
```

Comment:
Why have these specially for nondeterminism rather than passing through the types and converting that to the Rust types? 

If/when we add more types there will need to be two fields for every future type rather than just adding a branch to the converter

Reply:
Because this is a private FFI boundary and I only expose the most I need. I am trying to avoid overcomplicating the FFI code until there is a second type. I would not be surprised if that enum stays as only one item forever, heh, but it's much easier for Rust to build those abstractions than C.

Reply:
Sure, but, you've got to do the conversion either way. It's either the same amount of complicated, or if we ever add anything else, less complicated. You can just pass through an an enum or something, super simple, and less work to extend later.

Reply:
It's more complicated to do C arrays (not "super simple") and I have been lucky to avoid them so far because I can make expectations about how I can split single-value fields. But it sounds like my luck is running out.